packages/@ourworldindata/components/src/SharedDataPageConstants.ts

```diff
@@ -1,2 +1,3 @@
 export const DATAPAGE_SOURCES_AND_PROCESSING_SECTION_ID =
     "sources-and-processing" as const
+export const REUSE_THIS_WORK_SECTION_ID = "reuse-this-work"
```

Comment:
thanks for renaming the file! it really bugged me that I had given it such a generic name...